Source/VSProps/Base.props

```diff
@@ -19,7 +19,7 @@
       <ExternalTemplatesDiagnostics>true</ExternalTemplatesDiagnostics>
       <DisableAnalyzeExternal>true</DisableAnalyzeExternal>
       <TreatAngleIncludeAsExternal>true</TreatAngleIncludeAsExternal>
-
+      
       <WarningLevel>Level4</WarningLevel>
```

Comment:
This looks like an accident - did you mean to add whitespace here?

Reply:
No :p